test/index.test.ts

```diff
@@ -171,10 +170,12 @@ describe('cosmiconfig', () => {
       },
     };
 
-    const checkResult = (mock: jest.SpyInstance) => {
-      expect(mock.mock.calls.length).toEqual(1);
-      expect(mock.mock.calls[0].length).toEqual(1);
-      const { transform, loaders, ...explorerOptions } = mock.mock.calls[0][0];
+    const checkResult = (mock: jest.SpyInstance, instanceNum: number) => {
```

Comment:
The two `checkResult` functions in this file are largely identical except for different option objects; possibly worth extracting to a parameter and sharing the logic.